core/src/main/java/org/springframework/security/authentication/DelegatingReactiveAuthenticationManager.java

```diff
@@ -37,6 +40,20 @@ public class DelegatingReactiveAuthenticationManager implements ReactiveAuthenti
 
 	private final List<ReactiveAuthenticationManager> delegates;
 
+	private boolean continueOnError = false;
+
+	private static final Log logger = LogFactory.getLog(DelegatingReactiveAuthenticationManager.class);
```

Comment:
Please make the logger non-static, like so:

```java
private final Log logger = LogFactory.getLog(getClass());
```

Reply:
Fixed.